src/towers.effekt

```diff
@@ -0,0 +1,170 @@
+import mutable/array
+import src/benchmark
+import immutable/list
+
+//each list represents one smallerDisk.
+//TODO type alias, rename as "smallerDisk"
+//smallerDisk.next is the smallerDisk below => kellerstack
+type Disk = List[Int]
```

Comment:
Rename this to `Disks` or `Pile` or `Stack`. A single disk is an integer (its size).

Reply:
are we sure we want to deviate from the original?
it would change the semantics quite a lot and probably make it less comparable to the JS implementation of the benchmark.

Reply:
Yes, do it. The semantics will stay the same, we are merely using types to enforce preconditions.